### ISSUE TITLE
Configuration window not displaying properly when adding a Lora

### ISSUE THREAD

**Issue by DrFisura** [Acly/krita-ai-diffusion#1343]
First off, this wasn't an issue in previous versions of the plugin. Here's how it looks:

![image](https://github.com/user-attachments/assets/e37c4c27-57bf-4da3-8db9-2f8ec6fda64e)

It expands way too much to the right when adding at least one lora with the default window size that always comes at launch, which forces me to expand the window manually every time so that I can see the weights to the right and change them. It used to be correctly formatted in previous versions, where the weights were visible without having to expand the window manually. Here's how it looks without a lora:

![image](https://github.com/user-attachments/assets/3971fe7a-dd1e-44f6-8a4d-cd510ddb294a)

**Comment by Acly** [comment]
Any idea what might cause it? Do you have Lora in the folder which may have a very long name (including subfolders) that would require this kind of space?

**Comment by DrFisura** [comment]
You have some pretty strong intuition for this; I recently added four loras with very large names, and as soon as I have removed them right now, and updated the lora list (within version 1.27.1) the window format has immediately corrected just like old.
So maybe it has nothing to do with the version after all, but I've checked and I've had a lora as long as 155 characters (including the .safetensors extension) without it breaking the window format. You might want to look into this further, but I'd consider it a low priority, if you want to do something about the window width having more options.

Also, I've just checked right now: if I put back in another lora with a 156 characters long name, the field expands a little already. I'm not sure if the individual character length has any bearing on this.

I can live with slightly shorter lora names; the reason they got that long anyway is because I add related character tags in them so that I could potentially copy them easily if I needed to, but I rarely do

Oh and thank you again for your work and the functionality of this plugin
